velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -2848,24 +2848,6 @@ TEST_F(DateTimeFunctionsTest, dateParse) {
       dateParse("116", "%y+"), "Invalid format: \"116\" is malformed at \"6\"");
 }
 
-TEST_F(DateTimeFunctionsTest, dateFunctionVarchar) {
```

Comment:
Any reason to move this test?

Reply:
Whoops, that was accidental - didn't mean to commit. My bad, thanks for catching that. Reverted the change.